src/transforms/urlTransforms.ts

```diff
@@ -29,7 +29,7 @@ export async function transformBaseUrl(
   }
 
   return {
-    endpoint: endpoint,
+    endpoint: endpoint?.replace("http://", "https://"),
```

Comment:
I think we shouldn't replace urls, let users fix them in swagger or transforms. Or set the allowInsecureConnection flag

Reply:
I have fixed it by using the options.allowInsecureConnection flag in this logic.